resources/management/commands/resources_spaces_import.py

```diff
@@ -0,0 +1,79 @@
+import csv
+import os
+
+from django.core.management import BaseCommand, CommandError
+from django.utils import translation
+
+from resources.models import Resource, Unit, ResourceType, Purpose, ResourceGroup
+
+
+class Command(BaseCommand):
+    help = "Import resources via csv file. This is for importing resources for spaces"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "file_path", help="Path to the csv file",
+        )
+
+    @staticmethod
+    def get_authentication_type(authentication_type_fi):
+        if authentication_type_fi == 'ei mitään':
+            return 'none'
+        with translation.override("fi"):
+            name = [k for k in dict(Resource.AUTHENTICATION_TYPES).keys() if
```

Comment:
would make this a bit readable if the `dict(Resource.AUTHENTICATION_TYPES)` would be declared to its own variable: `authentication_types = dict(Resource.AUTHENTICATION_TYPES)`

Reply:
Thanks. I'll change it